backend/lcfs/web/api/fuel_supply/services.py

```diff
@@ -48,13 +47,15 @@ def fuel_type_row_mapper(self, compliance_period, fuel_types, row):
 
         row_data = dict(zip(column_names, row))
 
+        default_ci = row_data.get("default_carbon_intensity")
+        category_ci = row_data.get("category_carbon_intensity")
         fuel_category = FuelCategorySchema(
             fuel_category_id=row_data["fuel_category_id"],
             fuel_category=row_data["category"],
             default_and_prescribed_ci=(
-                round(row_data["default_carbon_intensity"], 2)
-                if row_data["fuel_type"] != "Other"
-                else default_ci.get(row_data["category"])
+                round(default_ci, 2)
```

Comment:
I think round returns the new object, so does this do anything? Should it be rounded_ci = round(default_ci, 2)?

Reply:
Passed to the schema.